src/api/cli/create_rawtx.go

```diff
@@ -531,14 +531,31 @@ func makeChangeOut(outs []wallet.UxBalance, chgAddr string, toAddrs []SendAmount
 		return nil, err
 	}
 
-	if haveChange {
-		outAddrs = append(outAddrs, mustMakeUtxoOutput(chgAddr, changeAmount, changeHours))
-	}
-
+	var spendCoinsAmt uint64
```

Comment:
does not need to be a var declaration, its only used inside of the loop each cycle